app/Http/Controllers/Admin/Dashboard/AdminDashboardController.php

```diff
@@ -7,36 +7,40 @@
 use App\Models\MentorProfile;
 use App\Models\MentorProfileMenteeProfile;
 use App\Models\User;
+use App\Presenters\MenteeProfilePresenter;
+use App\Presenters\MentorProfilePresenter;
 use Illuminate\Support\Facades\Redirect;
 use Inertia\Inertia;
 
 class AdminDashboardController
```

Comment:
I think there's a fair bit we can do here to clean this up but it might be a bit long winded to do this in a PR.  I might do a PR on this controller method separately and explain all the reasoning in the PR and we can discuss it

Reply:
Yes, please do, that would be great. Really interested in how it could be rewritten.